odoo/addons/__init__.py

```diff
@@ -1,25 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
If you remove it from here, can you also remove `odoo/upgrade/__init__.py`?

Reply:
We can move this part in an other pr.

Reply:
If you remove the init from addons, the reason to remove it from upgrade is the same (so I think they should be in one commit). But if you prefer to do it later, I'm ok with it too.